### ISSUE TITLE
Need to split time series data into train/val/test, but cannot shuffle

### ISSUE THREAD

**Issue by Cavernoso** [jfilter/split-folders#44]
As I am working with time series data, the information cannot be shuffled. 
Data volume is to big and therefore, I will use keras.train_on_batch since loading full dataset into RAM memory is not feasible. 
Spliting the data into train, test and validation folders would be helpfull. I am trying to use the split-folder library, but I could not identify how to avoid the shuffling process. Is it possible de-activate the shuffle?

**Comment by jfilter** [comment]
This is now supported. All split functions accept a `shuffle` parameter (defaults to `True`). Set `shuffle=False` to keep files in their sorted order — the first portion goes to train, the next to val, etc.

```python
import splitfolders

# Time series: split without shuffling
splitfolders.ratio("input_folder", output="output",
    ratio=(.8, .1, .1), shuffle=False)

splitfolders.fixed("input_folder", output="output",
    fixed=(100, 100), shuffle=False)

splitfolders.kfold("input_folder", output="output",
    k=5, shuffle=False)
```

From the CLI:
```
splitfolders --no-shuffle --ratio .8 .1 .1 -- input_folder
```
